setup_obs_virtual_cam_macos.sh

```
@@ -0,0 +1,244 @@
+        print_info "2. Téléchargez OBS Studio pour macOS"
+        print_info "3. Installez l'application"
+        print_info "4. Relancez ce script"
+        exit 1
+    fi
+fi
+
+# 2. Vérifier le plugin Virtual Camera
+print_info "Vérification du plugin Virtual Camera..."
+
+# OBS Virtual Camera est intégré depuis OBS 26.1
+OBS_VIRTUAL_CAM_AVAILABLE=false
```

Shellcheck (0.10.0)

[warning] OBS_VIRTUAL_CAM_AVAILABLE appears unused. Verify use (or export if used externally).

(SC2034)